### PR TITLE
More error logging for debug session

### PR DESCRIPTION
**Pull request by amit-chandak-unskript**
## Description
Added more debug when the remote session doesnt come up.

### Testing
Bad case

```
(base) root@awesome-awesome-runbooks-0:~# unskript-ctl.sh --debug --start --config $PWD/client-abc.ovpn
Error Occured while starting the Debug Session. Here are the logs from openvpn
===============================================================================================
Thu Dec 14 19:09:22 2023 Unrecognized option or missing or extra parameter(s) in /home/jovyan/client-abc.ovpn:18: block-outside-dns (2.4.7)
Thu Dec 14 19:09:22 2023 OpenVPN 2.4.7 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Mar 22 2022
Thu Dec 14 19:09:22 2023 library versions: OpenSSL 1.1.1f  31 Mar 2020, LZO 2.10
Thu Dec 14 19:09:22 2023 Outgoing Control Channel Encryption: Cipher 'AES-256-CTR' initialized with 256 bit key
Thu Dec 14 19:09:22 2023 Outgoing Control Channel Encryption: Using 256 bit message hash 'SHA256' for HMAC authentication
Thu Dec 14 19:09:22 2023 Incoming Control Channel Encryption: Cipher 'AES-256-CTR' initialized with 256 bit key
Thu Dec 14 19:09:22 2023 Incoming Control Channel Encryption: Using 256 bit message hash 'SHA256' for HMAC authentication
Thu Dec 14 19:09:22 2023 RESOLVE: Cannot resolve host address: tenant-staging-vp.alpha.unskript.io:443 (Name or service not known)
Thu Dec 14 19:09:22 2023 RESOLVE: Cannot resolve host address: tenant-staging-vp.alpha.unskript.io:443 (Name or service not known)
Thu Dec 14 19:09:22 2023 Could not determine IPv4/IPv6 protocol
Thu Dec 14 19:09:22 2023 SIGUSR1[soft,init_instance] received, process restarting
Thu Dec 14 19:09:22 2023 Restart pause, 5 second(s)
Thu Dec 14 19:09:27 2023 RESOLVE: Cannot resolve host address: tenant-staging-vp.alpha.unskript.io:443 (Name or service not known)
Thu Dec 14 19:09:27 2023 RESOLVE: Cannot resolve host address: tenant-staging-vp.alpha.unskript.io:443 (Name or service not known)
Thu Dec 14 19:09:27 2023 Could not determine IPv4/IPv6 protocol
Thu Dec 14 19:09:27 2023 SIGUSR1[soft,init_instance] received, process restarting
Thu Dec 14 19:09:27 2023 Restart pause, 5 second(s)

===============================================================================================
Stopped Active Debug session successfully
```

Good case

```
(base) root@awesome-awesome-runbooks-0:~# unskript-ctl.sh --debug --start --config $PWD/client-abc.ovpn
Successfully Started the Debug Session

```
### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: unskript-ctl/unskript_ctl_debug.py [hunk 1]
```diff
@@ -26,50 +26,72 @@ def start_debug(args):
...
+        print(f"ERROR: Unable to run the command {command}, error {e}")
+        return
 
     # Lets give few seconds for the subprocess to spawn
-    time.sleep(5)
+    try:
+        outs, errs = process.communicate(timeout=10)
+    except subprocess.TimeoutExpired:
+        # This is expected as the ovpn command needs to run indefinitely.
+        pass
```
Comment (jayasimha-raghavan-unskript) [inline unskript-ctl/unskript_ctl_debug.py:63]: What happens when timeout happens, May be it is a good idea to print some error message?

**Review by jayasimha-raghavan-unskript — COMMENTED** [review]
Please take care of the minor comment.

File: unskript-ctl/unskript_ctl_debug.py (hunk 1 above)
Comment (amit-chandak-unskript) [reply unskript-ctl/unskript_ctl_debug.py:63]: Timeout is expected as the process is running continuously

**Review by jayasimha-raghavan-unskript — APPROVED** [review]
LGTM

**Pull request merged**